addons/web/static/tests/fields/relational_fields_tests.js

```diff
@@ -9669,32 +9669,28 @@ QUnit.module('relational_fields', {
         // already available because they were in a inline view not already
         // opened, in a given configuration the change were applied ignoring
         // existing data, thus a line of a one2many field inside a one2many
-        // field could be duplicated unexplectedly
+        // field could be duplicated unexplectedly, or a posterious change lost
         assert.expect(5);
 
         var numUserOnchange = 0;
 
         this.data.user.onchanges = {
             partner_ids: function (obj) {
+                // simulate actual server onchange after save of modal with new record
```

Comment:
i think we should keep this test as it was, and add a new one (which is probably a slightly different version of this one), to ensure that the former bug doesn't come back